config/v3_3/config_test.go

```diff
@@ -142,23 +142,23 @@ func TestParse(t *testing.T) {
 		out out
 	}{
 		{
-			in:  in{config: []byte(`{"ignition": {"version": "3.2.0"}}`)},
+			in:  in{config: []byte(`{"ignition": {"version": "3.3.0"}}`)},
```

Comment:
We should document these test updates.

Also, it looks like v3_1 didn't get updated.